### DISCUSSION TITLE
Помогите донастроить.

### DISCUSSION THREAD

**Discussion by His88** [Anonym-tsk/nfqws-keenetic#24] [Q&A] [answered]
Добрый вечер, во-первых спасибо большое что проделали всю эту работу.

Я попробовал всё установить как написано в инструкции (скрин прилагаю). В итоге в конце пишет что "Instalation successful'' но ютуб так и не открывается.

П.С. мой уровень знаний - обычный пользователь, поэтому если какие-то уточняющие вопросы будут - напишите пожалуйста более развёрнуто.
![image](https://github.com/user-attachments/assets/68b0811e-5804-483d-96da-ee9c1f99a939)

**Comment by Anonym-tsk** [comment]
**Опишите проблему**
Подробно опишите что делали и что не работает.

**Модель маршрутизатора**
Укажите полную модель роутера и прошивку

**Провайдер**
Укажите вашего провайдера и тип подключения (ppp/ethernet/...)

**Выполните команды и приложите их вывод**
`/opt/etc/init.d/S51nfqws version`

`/opt/etc/init.d/S51nfqws restart`

`cat /opt/etc/nfqws/nfqws.conf`

`ps | grep nfqws`

`iptables-save | grep 200`

`sysctl net.netfilter.nf_conntrack_checksum`

**Reply by His88** [reply]
Подробно опишите что делали и что не работает:
1. Открыл вашу инструкцию на сайте и пошёл прямо пошагово по ней. Вроде как всё установилось без проблем (на скриншоте видно что установка успешно завершилась). Далее открываю браузер, ввожу адрес ютуба и страница вообще не загружается (как и до этого не загружалась после включения блокировок).

Укажите полную модель роутера и прошивку:
под прошивкой, наверное это Вы имели ввиду: Текущая версия KeeneticOS: [3.5.10]
Роутер Keenetic Extra II

Укажите вашего провайдера и тип подключения (ppp/ethernet/...)
провайдер вот: https://yoniti.ru/
тип подключения - не знаю честно говоря, там нет никаких аутентификаций PPPoE/L2TP - всё отключено, поэтому наверное ethernet


![image](https://github.com/user-attachments/assets/be54ba45-65d1-4fe0-b42e-aa9b62278611)

**Reply by Anonym-tsk** [reply, marked as answer]
Вы все сделали правильно, кроме того, что выбрали архитектуру arm. Ваша вероятнее всего mipsel. Переустановите скрипты, должно заработать.

**Reply by His88** [reply]
да, теперь ютуб заработал. Спасибо Вам за помощь, Вы меня очень выручили.
